skeleton/default.nix

```diff
@@ -15,14 +15,38 @@
   }
 }:
 with obelisk;
-project {} ({ pkgs, ... }: {
+let args = {
+  # You must accept the Android Software Development Kit License Agreement at
+  # https://developer.android.com/studio/terms in order to build Android apps.
+  # Uncomment this to indicate your acceptance:
+  android_sdk_accept_license = true;
+  allowUnfree = true;
+}; in
+
+project args ({ pkgs, ... }: {
   name = "skeleton";
+  android = {
+    executable = ps: exes: (exes (ps.frontend)).frontend;
+    executableName = "obelisk-skeleton";
+    applicationId = "systems.obsidian.obelisk.examples.minimal";
+    displayName = "Obelisk Minimal Example";
+  };
+  ios = {
+    executable = ps: exes: (exes (ps.reflex-todomvc)).reflex-todomvc;
+    executableName = "reflex-todomvc";
+    bundleIdentifier = "systems.obsidian.obelisk.examples.minimal";
+    bundleName = "Obelisk Minimal Example";
+  };
+  web = {
+    executable = ps: exes: (exes (ps.frontend)).frontend;
+  };
   extraArgs = {
     staticFiles = ./static;
   };
   src = ./.;
   inputThunks = pkgs.obsidianCompilers.thunkSets.common ++ [
     pkgs._dep.source.aeson-1541
+    pkgs._dep.source.android-activity
```

Comment:
Version on hackage is too old and a release wouldn't fix it until we bump index state

Reply:
makes sense!